google/field_helpers_test.go

```diff
@@ -1,36 +1,84 @@
 package google
 
-import "testing"
+import (
+	"testing"
+)
 
-func TestParseNetworkFieldValue(t *testing.T) {
+func TestParseGlobalFieldValue(t *testing.T) {
+	const resourceType = "networks"
 	cases := map[string]struct {
-		Network              string
+		FieldValue           string
 		ExpectedRelativeLink string
+		ExpectedError        bool
+		IsEmptyValid         bool
+		ProjectSchemaField   string
+		ProjectSchemaValue   string
 		Config               *Config
 	}{
 		"network is a full self link": {
-			Network:              "https://www.googleapis.com/compute/v1/projects/myproject/global/networks/my-network",
+			FieldValue:           "https://www.googleapis.com/compute/v1/projects/myproject/global/networks/my-network",
 			ExpectedRelativeLink: "projects/myproject/global/networks/my-network",
 		},
 		"network is a relative self link": {
-			Network:              "projects/myproject/global/networks/my-network",
+			FieldValue:           "projects/myproject/global/networks/my-network",
 			ExpectedRelativeLink: "projects/myproject/global/networks/my-network",
 		},
 		"network is a partial relative self link": {
-			Network:              "global/networks/my-network",
+			FieldValue:           "global/networks/my-network",
 			ExpectedRelativeLink: "projects/default-project/global/networks/my-network",
 			Config:               &Config{Project: "default-project"},
 		},
 		"network is the name only": {
-			Network:              "my-network",
+			FieldValue:           "my-network",
 			ExpectedRelativeLink: "projects/default-project/global/networks/my-network",
 			Config:               &Config{Project: "default-project"},
 		},
+		"network is the name only and has a project set in schema": {
+			FieldValue:           "my-network",
+			ExpectedRelativeLink: "projects/schema-project/global/networks/my-network",
```

Comment:
nit: I feel like it would be the teeniest bit easier to read if you put FieldValue, ProjectSchemaField, and ProjectSchemaValue before ExpectedRelativeLink

Reply:
done